core/store/src/db/rocksdb.rs

```diff
@@ -528,7 +528,12 @@ fn rocksdb_column_options(col: DBCol, store_config: &StoreConfig, temp: Temperat
     opts
 }
 
-fn set_compression_options(opts: &mut Options) {
+fn set_compression_options(opts: &mut Options, disable_compression: bool) {
```

Comment:
(unrelated to this line)
Please mention somewhere in fork-network comments, that you recommend disabling rocksdb compression and what kind of speed up can be expected.

Reply:
Actually, I tried without it and there's no real difference. Compression spins up more threads to do compression, using up more CPU, but there's enough CPU with 8 cores anyway.